### ISSUE TITLE
revise `role` vs `hasOrganisationRole`

### ISSUE THREAD

**Issue by VladimirAlexiev** [Interoperable-data/ERA-Ontology-3.1.0#74]
This pattern is unclear to me:
```ttl
era:hasOrganisationRole rdf:type owl:ObjectProperty ;
                        rdfs:domain era:OrganisationRole ;
                        rdfs:range <http://www.w3.org/2004/02/skos/core#Concept> ;
                        rdfs:comment "The role of the body"@en ;
                        rdfs:label "has organisation role"@en .
era:role rdf:type owl:ObjectProperty ;
         rdfs:domain era:Body ;
         rdfs:range era:OrganisationRole ;
         rdfs:label "role"@en .
```
- the comment "The role of the body" is wrong because the domain is not `Body`
- The prop name `era:hasOrganisationRole` is wrong because it should be **incoming to** `OrganisationRole` not **outgoing from** it
- Is `OrganisationRole` supposed to be specific to a `Body`, eg would it have time validity information?
  - If so, document it and add these extra props to `OrganisationRole`
  - If not, then isn't it enough to have an `OrganisationRole` ConceptScheme (rather than class) and keep only one of these 2 props?

Note: none of these props and class have any instances on https://virtuoso.ecdp.tech.ec.europa.eu/sparql/ yet.

**Comment by Interoperable-data** [comment]
Yes, this is a generic pattern for ORG and will contain time validity information, and more properties, like certificates in some cases.

**Comment by VladimirAlexiev** [comment]
@Interoperable-data can you summarize what you did? Paste the turtle blocks relevant to this issue. thanks!!
